frontend/src/App.tsx

```diff
@@ -61,6 +62,12 @@ const App = (): React.ReactElement => {
             <Switch>
               <Route exact path={Routes.LOGIN_PAGE} component={Login} />
               <Route exact path={Routes.SIGNUP_PAGE} component={Signup} />
+              <PrivateRoute
```

Comment:
This shouldn't be a private route, since people that aren't authenticated yet should be able to access the forgot password flow

Reply:
@jerry-cheng5 yeah makes sense. made the fix